docs/auto-enable.md

```diff
@@ -0,0 +1,107 @@
+# Make `pg_stat_monitor` automatically accessible for every newly created database
+
+In PostgreSQL, a database is created by copying the [system template database](https://www.postgresql.org/docs/current/manage-ag-templatedbs.html) `template1`. To automatically create the `pg_stat_monitor` extension for every newly created database, you can do the following:
+
+* modify the `template1` template database and create the extension for it. Then all databases you create have the `pg_stat_monitor` view available.
```

Comment:
create or enable the extension for it?

ETA: In reading a bit further down, the command is "create extension" but I think we're actually enabling the extension in the template DB _for the next created database_ at least that is how it makes sense to me.